languages/elixir/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,55 @@
+While preparing to bake cookies for your friends, you have found that you have to convert some of the ingredients used in the recipe for your friends. Being only familiar with the metric system, you need to come up with a way to convert common US baking measurements to millilitres (mL) for your own ease.
+
+Use this conversion chart for your solution:
+
+| Unit to convert | volume | in millilitres (mL) |
+| --------------- | ------ | ------------------- |
+| mL              | 1      | 1                   |
+| US cup          | 1      | 240                 |
+| US fluid ounce  | 1      | 30                  |
+| US teaspoon     | 1      | 5                   |
+| US tablespoon   | 1      | 15                  |
+
+Being a talented programmer in training, you decide to use millilitres as a transition unit to facilitate the conversion from any unit listed to any other (even itself)
+
+### 1. Implement the `get_volume/1` function
```

Comment:
Could you make it so that the task headers in this file (and the corresponding headers in the `hints.md` file) describe what is being done instead of how? (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsinstructionsmd)). So something like: 

```suggestion
### 1. Get the volume
```

Or something like that.

Reply:
Ya, I might need to change a few of the existing exercises... :disappointed_relieved:

Reply:
Sorry for not catching those!